### DISCUSSION TITLE
MMFF94s_new parameters

### DISCUSSION THREAD

**Discussion by hypnopump** [rdkit/rdkit#6765] [Ideas]
This paper ["Accuracy evaluation and addition of improved dihedral parameters for the MMFF94s"](https://jcheminf.biomedcentral.com/counter/pdf/10.1186/s13321-019-0371-6.pdf) from 2019 motivated a reparametrization of the `MMFF94s` forcefield such that bioactive conformations could be retrieved more often (a proxy for a better forcefield); and it implemented the method in DataWarrior. It would be nice to port those parameters and have them available in the RDKit

**Comment by bp-kelley** [comment]
The parameters would be pretty simple to add, I'm not convinced that it will perform better than ETKDG though for conformation generation.  Might be a nice evaluation paper for someone though :)
